kolibri/plugins/management/assets/src/device_management/views/manage-content-page/task-status.vue

```diff
@@ -15,16 +15,33 @@
 
   /* eslint-env node */
 
-  import * as manageContentActions from '../../state/manageContentActions';
-  import * as actions from '../../state/actions';
-  import * as constants from '../../constants';
+  import { refreshChannelList } from '../../state/actions/manageContentActions';
+  import { cancelTask } from '../../state/actions/taskActions';
+  import { TaskTypes, TaskStatuses } from '../../../constants';
   import logger from 'kolibri.lib.logging';
   const logging = logger.getLogger(__filename);
-  const TaskTypes = constants.TaskTypes;
-  const TaskStatuses = constants.TaskStatuses;
   import kButton from 'kolibri.coreVue.components.kButton';
   export default {
-    name: 'contentPage',
+    name: 'taskStatus',
+    components: { kButton },
+    props: {
```

Comment:
Again, would prefer to avoid this kind of linting that is not following a repo guideline, and serves to obscure actual changes made.